src/foundry/foundry.js/clientDocuments/combat.d.ts

```diff
@@ -1,4 +1,6 @@
-// TODO
+import { ConfiguredDocumentClass } from '../../../types/helperTypes';
+import { DocumentModificationOptions } from '../../common/abstract/document.mjs';
+
 declare global {
   /**
    * The client-side Combat document which extends the common BaseCombat abstraction.
```

Comment:
The JSDoc in foundry.js seems to be incorrect, it's just copied over from BaseCombat. Please consider creating an issue at foundry GitLab there isn't already a corresponding one.

It looks like we are actually using the documentation that _should_ be there, so on our side, I think we're ok.

Reply:
That's not correct, `Combat` has an additional line in `foundry.js`. Yes the first line is copied, but it expanded upon.